commandline/src/main/java/tech/pegasys/web3signer/commandline/CommandlineParser.java

```diff
@@ -152,7 +115,12 @@ private int handleExecutionException(
       errorWriter.println("Failed to initialize Web3Signer");
       errorWriter.println("Cause: " + ex.getMessage());
     }
-    commandLine.usage(outputWriter);
+
+    if (baseCommand.getLogLevel() != null
+        && Level.DEBUG.isMoreSpecificThan(baseCommand.getLogLevel())) {
+      ex.printStackTrace(errorWriter);
```

Comment:
Why not print the error using the same as done above? i.e.
```
errorWriter.println("Failed to initialize Web3Signer");
errorWriter.println("Cause: " + ex.getMessage());
```

Reply:
good question - I've cleaned this up, hopefully better.